Detectors/TRD/base/test/testCoordinateTransforms.cxx

```diff
@@ -43,7 +44,13 @@ void testRCPoint(double calculatedPoint, double predictedPoint)
 
 BOOST_AUTO_TEST_CASE(LocaltoRCTest)
 {
+  // init ALICE geometry --> needed to instantiate TRD Geometry instance
+  auto& ccdbmgr = o2::ccdb::BasicCCDBManager::instance();
+  ccdbmgr.get<TGeoManager>("GLO/Config/GeometryAligned");
+
   auto mGeo = o2::trd::Geometry::instance();
+  BOOST_CHECK(mGeo);
+
   mGeo->createPadPlaneArray();
   mGeo->createClusterMatrixArray();
```

Comment:
Actually, just this line here should be removed which causes the error to be printed. The cluster matrix array is not used in this test so this is not the issue

Reply:
Yes, I agree. Will close this PR.

Reply:
TRD should then remove the line `mGeo->createClusterMatrixArray`